packages/editor-ui/src/components/Sticky.vue

```diff
@@ -239,6 +299,17 @@ export default defineComponent({
 			return this.uiStore.isActionActive('workflowRunning');
 		},
 	},
+	mounted() {
+		// Initialize the node
+		if (this.data !== null) {
+			try {
+				this.addNode(this.data);
+			} catch (error) {
+				// This breaks when new nodes are loaded into store but workflow tab is not currently active
+				// Shouldn't affect anything
+			}
+		}
+	},
```

Comment:
same question here about code duplication.

Reply:
We'll probably need a composable after we rewrite.